pkg/manager/member/tikv_member_manager.go

```diff
@@ -511,6 +511,37 @@ func volumeClaimTemplate(r corev1.ResourceRequirements, metaName string, storage
 	}
 }
 
+func transformTiKVConfigMap(src []byte, tc *v1alpha1.TidbCluster) string {
```

Comment:
can we also add a link to tikv change PR on these fields? then people can know why this function is required

Reply:
updated.